docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -188,9 +188,9 @@ Using source assets has a few advantages over having the code inside of an asset
 - **Dagster can use data-loading code factored into an <PyObject object="IOManager" /> to load the contents of the source asset**.
 - **Asset dependencies can be written in a consistent way,** independent of whether they're downstream from a source asset or a derived asset. This makes it easy to swap out a source asset for a derived asset and vice versa.
 
-#### Non-argument dependencies
+#### Defining dependencies that don't load data
```

Comment:
wasn't sure what to rename this - open to suggestions!

Reply:
Chatted with Tim about this heading/section - some initial thoughts:

- Should this copy explicitly state that this has to do with I/O managers? Or no, since we're trying to simplify concepts?
- One thing this section is missing is when you'd use this instead of just normal dependencies. Ex: Setting up the order of assets, data is too big to load, etc.
- Title may actually be fine if we add some of the why/when to this

Thoughts?